docs/rpc/README.md

```diff
@@ -6,47 +6,106 @@ linkTitle: RPC
 >     allowing for a phased rollout of the stable semantic conventions.
->   * The default behavior (in the absence of one of these values) is to continue
+>   - The default behavior (in the absence of one of these values) is to continue
 >     emitting whatever version of the old experimental RPC conventions
 >     the instrumentation was emitting previously.
->   * Note: `rpc/dup` has higher precedence than `rpc` in case both values are present
-> * SHOULD maintain (security patching at a minimum) their existing major version
+>   - Note: `rpc/dup` has higher precedence than `rpc` in case both values are present
+> - SHOULD maintain (security patching at a minimum) their existing major version
 >   for at least six months after it starts emitting both sets of conventions.
-> * MAY drop the environment variable in their next major version and emit only
+> - MAY drop the environment variable in their next major version and emit only
 >   the stable RPC conventions.
 
-Semantic conventions for RPC are defined for the following signals:
+The RPC Semantic conventions are designed to cover the logical operation of invoking an operation (function) to run in a remote process.
+This remote process call will often travel across the network to reach it's destination.
 
-* [RPC Spans](rpc-spans.md): Semantic Conventions for RPC client and server *spans*.
-* [RPC Metrics](rpc-metrics.md): Semantic Conventions for RPC *metrics*.
+The details of this invocation can be captured using the following signals:
 
-Technology specific semantic conventions are defined for the following RPC systems:
+- [RPC Spans](rpc-spans.md): Semantic Conventions for RPC client and server *spans*.
+- [RPC Metrics](rpc-metrics.md): Semantic Conventions for RPC *metrics*.
 
-* [Connect](connect-rpc.md): Semantic Conventions for *Connect RPC*.
-* [gRPC](grpc.md): Semantic Conventions for *gRPC*.
-* [JSON-RPC](json-rpc.md): Semantic Conventions for *JSON-RPC*.
+These generic conventions can be extended based on the [RPC Protocol](#what-is-a-rpc-protocol) being used with the following protocols defined:
+
+- [gRPC](grpc.md)
+- GWT
+- [Http](/docs/http/README.md)
+- Triple
+- Twirp
+- SOAP
+- [Connect RPC](connect-rpc.md)
+
+They can also be extended based on the [RPC Framework](#what-is-a-rpc-framework) being used with the following frameworks defined:
+
+- Apache Dubbo
+- Dapr
+- [Connect RPC](connect-rpc.md)
+- WCF
+
+However should the process being invoked be a member of a more specific domain such as those below,
+the corresponding conventions should be followed:
+
+- [Database](/docs/database/README.md)
+- [Generative AI](/docs/gen-ai/README.md)
+- [Messaging](/docs/messaging/README.md)
+
+If your focus is on the network layer calls rather than the logical calls being made,
+these RPC documents are not for you but instead what you are after is described via the below conventions:
+
+- Http
+
+## RPC Protocol vs RPC Framework
```

Comment:
do we need to describe this at all? The difference is very subjective and nothing(?) in the semconv depends on this terminology being established